vita3k/config/src/config.cpp

```diff
@@ -227,11 +227,11 @@ ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths)
         cfg.pkg_zrif = std::move(command_line.pkg_zrif);
         return QuitRequested;
     }
-    if (command_line.load_config || command_line.config_path != root_paths.get_base_path()) {
+    if (command_line.load_config || command_line.config_path != root_paths.get_config_path()) {
         if (command_line.config_path.empty()) {
-            command_line.config_path = root_paths.get_base_path();
+            command_line.config_path = root_paths.get_config_path();
         } else {
-            if (parse(command_line, command_line.config_path, root_paths.get_base_path()) != Success)
+            if (parse(command_line, command_line.config_path, root_paths.get_pref_path()) != Success)
```

Comment:
```suggestion
            if (parse(command_line, command_line.config_path, root_paths.get_config_path()) != Success)
```

Reply:
Doing this would change the default pref_path to config_path anytime someone used the command line to change the config_path.

Reply:
eu, i have no understund this code totaly no relate pref path ? is just relate base path

Reply:
What is said, is so why before is base path ?